pkgs/applications/misc/gxmessage/default.nix

```diff
@@ -0,0 +1,20 @@
+{stdenv, fetchurl, gnome3, intltool, pkgconfig, texinfo}:
+
+stdenv.mkDerivation rec {
+  name = "gxmessage";
+  version = "3.4.3";
+
```

Comment:
Well, sure. But won't pushing a new commit trigger a travis job? That failure in travis looks unrelated to this PR, so I guess there's no reason not to push.